gravitee-apim-console-webui/src/management/api/notification-settings/notifications-list/notifications-list-api.component.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Inject, OnInit } from '@angular/core';
+import { Subject } from 'rxjs';
+
+import { NotificationSettingsService } from '../../../../services-ngx/notification-settings.service';
+import { UIRouterStateParams } from '../../../../ajs-upgraded-providers';
+import { NotificationsServices } from '../../../../components/notifications/notifications-list/notifications-list.component';
+
+@Component({
+  selector: 'notification-settings-list-api',
+  template: require('./notifications-list-api.component.html'),
+})
+export class NotificationsListApiComponent implements OnInit {
```

Comment:
```suggestion
export class ApiNotificationSettingsListComponent implements OnInit {
```
What do you think about adding the "domain" first (like other existing components)? and keep NotificationSettings instead of notifications 

IMHO this page does not display notifications but their settings

Reply:
done